src/ui/components/Library/Team/View/NewTestRuns/TestRunsPage.tsx

```diff
@@ -98,16 +98,18 @@ function TestRunsContent() {
         <Panel minSize={20} order={1}>
           <div className="flex h-full w-full flex-col gap-4 overflow-hidden rounded-xl bg-bodyBgcolor p-2">
             <div className="flex flex-col gap-2">
-              <div className="grid w-full grid-cols-3 gap-2 bg-bodyBgcolor">
+              <div className="grid w-full grid-cols-3 gap-2 overflow-hidden bg-bodyBgcolor">
                 <div
                   className={dropdownStyles.dropdownTrigger}
                   data-test-id="TestRunsPage-ResultFilter-DropdownTrigger"
                   onClick={onClickStatusFilter}
                   onKeyDown={onKeyDownStatusFilter}
                   tabIndex={0}
                 >
-                  {filterByStatus === "all" ? "All runs" : "Only failures"}
-                  <Icon className="h-5 w-5" type="chevron-down" />
+                  <div className="overflow-hidden overflow-ellipsis whitespace-pre">
```

Comment:
Do you mean `whitespace-pre` here instead of `nowrap`? This can be done with the `truncate` tailwind class which includes the 3 CSS rules to overflow with ellipsis.

Reply:
TIL…